tests/backend/test_signup_requests.py

```
@@ -0,0 +1,80 @@
+    ],
+)
+def test_normalise_payload_rejects_invalid(payload):
+    with pytest.raises(signup_requests.SignupValidationError):
+        signup_requests.normalise_payload(payload)
+
+
+def test_create_signup_request_persists_record(tmp_path):
+    now = datetime(2026, 6, 15, 12, 0, 0, tzinfo=timezone.utc)
+    store = signup_requests.signup_requests_dir(tmp_path)
+
+    record, token = signup_requests.create_signup_request(
```

Ruff (0.15.15)

[warning] Unpacked variable `token` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)